base/deprecated.jl

```diff
@@ -1499,4 +1499,13 @@ end
 # Calling promote_op is likely a bad idea, so deprecate its convenience wrapper promote_eltype_op
 @deprecate promote_eltype_op(op, As...) promote_op(op, map(eltype, As)...)
 
+function include(_path::AbstractString...)
+    depwarn("multi-argument `include` is deprecated, use multiple `include` calls instead", :include)
+    local result
+    for path in _path
+        result = include(path)
+    end
+    result
+end
```

Comment:
Perhaps the deprecation of multi-argument `include` should be in #19909 instead?